### PR TITLE
#159053962 Create Articles Rating

### PR DESCRIPTION
**Pull request by janetnim**
#### What does this PR do?
Has the average rating to articles

#### Description of tasks to be completed
Add rating to an existing article with a slug

#### How should this be manually tested?
Clone the repo with the following command on your terminal
```
git clone https://github.com/andela/ah-titans/.git
```
Install your virtual environment and activate it then pip install the requirements with the commands:
```
virtualenv venv
```
```
source env/bin/activate
```
```
pip install -r requirements.txt
```
Run the server
```
python manage.py runserver
```
Add rating to an article with a slug in postman as shown below:

screenshot 2018-08-03 at 10 07 48

 #### Any background context you want to provide?
Templates for article rating:

screen shot 2018-08-03 at 09 31 21

screen shot 2018-08-03 at 09 30 15

#### What are the relevant Pivotal Tracker Stories?
[#159053962](https://www.pivotaltracker.com/story/show/159053962)

### CODE REVIEW

File: authors/apps/articles/serializers.py [hunk 1]
```diff
@@ -17,11 +17,12 @@ class ArticleSerializer(serializers.ModelSerializer):
     created_at = serializers.DateTimeField(read_only=True)
     updated_at = serializers.DateTimeField(read_only=True)
     author = ProfileSerializer(read_only=True)
+    rating = serializers.IntegerField(required=False)
```
Comment (ronaldndirangu) [inline authors/apps/articles/serializers.py:20]: I think you should also add a field for the total number of people who rated the article. We might need it in the future.

File: authors/templates/my-article.html [hunk 2]
```diff
@@ -117,4 +129,4 @@ <h5>Comments:</h5>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script>
 </body>
 
-</html>
+</html>
```
Comment (petermwash) [inline authors/templates/my-article.html]: Add an end of line

File: authors/templates/my-article.html [hunk 3]
```diff
@@ -117,4 +129,4 @@ <h5>Comments:</h5>
     <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0-beta/js/materialize.min.js"></script>
```
Comment (dcDalin) [inline authors/templates/my-article.html]: Consider using the release version of materialize.min.js, not the beta version.

File: authors/apps/articles/serializers.py (hunk 1 above)
Comment (janetnim) [reply authors/apps/articles/serializers.py:20]: @ronaldndirangu Ronald Thanks I've done that

**Comment by chepmanu** [comment]
Nice rating implementation, looks good to me.

**Review by ridgekimani — APPROVED** [review]

**Pull request merged**